products/rancher.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Rancher
+category: service
```

Comment:
```suggestion
category: server-app
```

`service` is reserved for managed service for now (maybe not the best name). We should be using the same tag as https://endoflife.date/argo-cd for now.